blocks/editable/style.scss

```diff
@@ -117,6 +118,7 @@ input.editable-format-toolbar__link-input {
 	position: relative;
 
 	&:after {
-		@include long-content-fade( $size: 40% );
+
```

Comment:
Similarly, what's the guideline for the newline here? Doesn't follow my inferred guideline from my previous comment.